### ISSUE TITLE
Web standards - Comms 1 

### ISSUE THREAD

**Issue by nfee006** [asc-csa/ckanext-asc-csa#137]
Several WCAG errors 
1.    There are two Search Canada.ca 
[WCAG21 2.4.6 (AA)] Form control labels must be unique: See matching tag on line: 135)
`<label for="wb-srch-q" class="wb-inv">Search Canada.ca</label>`

2.    There are quotes missing from the action attribute 
Invalid characters found in the 'action' attribute's value starting at character: 7
`<form action=https://www.canada.ca/en/sr/srb.html method="post" name="cse-search-box" role="search">`

3.    There are quotes missing from the href attribute
Invalid characters found in the 'action' attribute's value starting at character: 7

`<a href=https://www.canada.ca/en/contact.html>`
`<a href=https://www.canada.ca/en/government/dept.html>`
`<a href=https://www.canada.ca/en/government/publicservice.html>`
etc... (see email)

4.    There is a second error where the last character of the href is not in quotes and therefore is considered the end of the a tag, therefore the a tag closing is not associated with any starting a tag. 
`<a href=https://open.canada.ca/en/>Open government`
No matching start tag found: `</a>`

@coleweusthem

**Comment by coleweusthem** [comment]
(3) should be fixed now

**Comment by coleweusthem** [comment]
fixed 1) looks like they changed the canada.ca header a bit, so I replaced the search with what they have now

**Comment by coleweusthem** [comment]
fixed 2) as part of 1

**Comment by coleweusthem** [comment]
Ok I think that's everything if you want to double check.

**Comment by nfee006** [comment]
I'll do so shortly - thanks!

**Comment by nfee006** [comment]
@coleweusthem Still seeing issue 3 
For reference, the header (with quotes): 
![image](https://user-images.githubusercontent.com/11013061/127190190-bb83136a-bfb7-4258-867f-64a9be92967d.png)

The footer (missing quotes):
![image](https://user-images.githubusercontent.com/11013061/127190316-efdc9b70-9614-4d92-88d3-f632c688202e.png)

**Comment by nfee006** [comment]
@coleweusthem - just checking, did you see this?

**Comment by coleweusthem** [comment]
<img width="1329" alt="Screen Shot 2021-07-28 at 10 04 54 AM" src="https://user-images.githubusercontent.com/46227765/127327059-d827e330-9ad4-444c-80a8-3aff8f36db5e.png">
Dash's templating system doesn't want to let me stick in double quotes, but the above with single quotes should be fine, I'm not sure why you aren't seeing it. I'll double-check I pushed the change

**Comment by emilinecsa** [comment]
What's left in this task? Was #3 fixed? What about #4? My understanding is that #4 still need to be done. Am I right?

**Comment by nfee006** [comment]
@emilinecsa I believe these were all addressed. I can't replicate them.

**Comment by emilinecsa** [comment]
@nfee006 Thank you!
